code/game/turfs/hybrisa.dm

```diff
@@ -0,0 +1,717 @@
+	name = "ribbed metal walls"
+	icon_state = "strata_ribbed_outpost_"
+	desc = "A thick and chunky metal wall covered in jagged ribs."
+	walltype = WALL_STRATA_OUTPOST_RIBBED
+	damage_cap = HEALTH_WALL_REINFORCED
+/turf/closed/wall/hybrisa/colony/hospital/reinforced/hull
+	hull = TRUE
+	icon_state = "strata_hull"
+	desc = "A thick and chunky metal wall that is, just by virtue of its placement and imposing presence, entirely indestructible."
+
+// Offices
+
+/turf/closed/wall/hybrisa/colony/office/ribbed //this guy is our reinforced replacement
+	name = "ribbed metal walls"
+	icon = 'icons/turf/walls/hybrisa_offices_colonywall.dmi'
+	icon_state = "strata_ribbed_outpost_"
+	desc = "A thick and chunky metal wall covered in jagged ribs."
+	walltype = WALL_STRATA_OUTPOST_RIBBED
+	damage_cap = HEALTH_WALL_REINFORCED
+/turf/closed/wall/hybrisa/colony/office
+	name = "bare metal walls"
+	icon = 'icons/turf/walls/hybrisa_offices_colonywall.dmi'
+	icon_state = "strata_bare_outpost_"
+	desc = "A thick and chunky metal wall. The surface is barren and imposing."
+	walltype = WALL_STRATA_OUTPOST_BARE
+/turf/closed/wall/hybrisa/colony/office/reinforced
+	name = "ribbed metal walls"
+	icon_state = "strata_ribbed_outpost_"
+	desc = "A thick and chunky metal wall covered in jagged ribs."
+	walltype = WALL_STRATA_OUTPOST_RIBBED
+	damage_cap = HEALTH_WALL_REINFORCED
+/turf/closed/wall/hybrisa/colony/office/reinforced/hull
+	hull = 1
+	icon_state = "strata_hull"
+	desc = "A thick and chunky metal wall that is, just by virtue of its placement and imposing presence, entirely indestructible."
+
+// Engineering
+
+/turf/closed/wall/hybrisa/colony/engineering/ribbed //this guy is our reinforced replacement
+	name = "ribbed metal walls"
+	icon = 'icons/turf/walls/hybrisa_engineering_wall.dmi'
+	icon_state = "strata_ribbed_outpost_"
+	desc = "A thick and chunky metal wall covered in jagged ribs."
+	walltype = WALL_STRATA_OUTPOST_RIBBED
+	damage_cap = HEALTH_WALL_REINFORCED
+/turf/closed/wall/hybrisa/colony/engineering
+	name = "bare metal walls"
+	icon = 'icons/turf/walls/hybrisa_engineering_wall.dmi'
+	icon_state = "strata_bare_outpost_"
+	desc = "A thick and chunky metal wall. The surface is barren and imposing."
+	walltype = WALL_STRATA_OUTPOST_BARE
+/turf/closed/wall/hybrisa/colony/engineering/reinforced
+	name = "ribbed metal walls"
+	icon_state = "strata_ribbed_outpost_"
+	desc = "A thick and chunky metal wall covered in jagged ribs."
+	walltype = WALL_STRATA_OUTPOST_RIBBED
+	damage_cap = HEALTH_WALL_REINFORCED
+/turf/closed/wall/hybrisa/colony/engineering/reinforced/hull
+	hull = TRUE
+	icon_state = "strata_hull"
+	desc = "A thick and chunky metal wall that is, just by virtue of its placement and imposing presence, entirely indestructible."
+
+// Space-Port
+
+/turf/closed/wall/hybrisa/spaceport
+	name = "metal wall"
+	icon = 'icons/turf/walls/hybrisa_spaceport_walls.dmi'
+	icon_state = "metal"
+	walltype = WALL_METAL
+/turf/closed/wall/hybrisa/spaceport_reinforced
```

Comment:
Any reason why not just `/turf/closed/wall/hybrisa/spaceport/reinforced` so you can just use inheritance and only change what needs to change?